components/generic/HTMLEmbed.vue

```diff
@@ -18,18 +30,42 @@
 
   export default {
     name: 'OEmbedMedia',
+
     components: {
       AlertMessage
     },
+
     props: {
       html: {
         type: String,
         default: ''
       },
+      provider: {
+        type: String,
+        default: ''
+      },
+      height: {
+        type: Number,
+        default: null
+      },
+      width: {
+        type: Number,
+        default: null
+      },
       error: {
         type: [String, Error],
         default: ''
       }
+    },
+
+    computed: {
+      aspectRatio() {
```

Comment:
This does not return the aspect ratio, so should be renamed.

Perhaps `heightAsPercentOfWidth` returning `(this.height * 100) / this.width`, with the `padding-bottom` moved into the style attribute on the youtube embed div.